crates/revm/Cargo.toml

```diff
@@ -24,6 +24,7 @@ tokio = { version = "1.28", features = ["rt-multi-thread", "macros"], optional =
 ethers-providers = { version = "2.0", optional = true }
 ethers-core = { version = "2.0", optional = true }
 futures = { version = "0.3.27", optional = true }
+rust-kzg = "0.0.0"
```

Comment:
Is this used?

Reply:
No i was using it at one point when writing the point verification by hand prior to using the EF bindings.